pkg/liqo-controller-manager/namespaceoffloading-controller/clusterselector.go

```diff
@@ -110,3 +110,13 @@ func matchVirtualNodeSelectorTerms(ctx context.Context, cl client.Client, virtua
 	}
 	return k8shelper.MatchNodeSelectorTerms(n, selector)
 }
+
+// MatchNodeSelectorTerms checks if the node match the node selector.
+func MatchNodeSelectorTerms(node *corev1.Node, selector *corev1.NodeSelector) (bool, error) {
```

Comment:
Do we need this function? It seems to only be an empty wrapper

Reply:
Right, I can use MatchVirtualNodeSelectorTerms